### PR TITLE
Update random_crop_and_zoom.py

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Update RandomResizedCrop to RandomCropAndResize

### CODE REVIEW

File: examples/layers/preprocessing/classification/random_crop_and_zoom.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""random_resized_crop_demo.py.py shows how to use the RandomResizedCrop
+"""random_crop_and_resize.py shows how to use the RandomCropAndResize
```
Comment (divyashreepathihalli) [inline examples/layers/preprocessing/classification/random_crop_and_zoom.py:15]: Lets rephrase it as " This demo example shows how to use `RandomResizedCrop`

**Review by divyashreepathihalli — CHANGES_REQUESTED** [review]
let us update the file name to random_crop_and_resize_demo.py

File: examples/layers/preprocessing/classification/random_crop_and_zoom.py (hunk 1 above)
Comment (divyashreepathihalli) [reply examples/layers/preprocessing/classification/random_crop_and_zoom.py]: Thank you - let us update the file name to random_crop_and_resize_demo.py

Comment (sachinprasadhs) [reply examples/layers/preprocessing/classification/random_crop_and_zoom.py]: Done

**Review by divyashreepathihalli — APPROVED** [review]

**Comment by divyashreepathihalli** [comment]
LGTM! Thank you! Merged!

**Pull request merged**